x-pack/plugins/cases/public/components/cases_context/index.tsx

```diff
@@ -149,16 +152,18 @@ export const CasesProvider: React.FC<{ value: CasesContextProps }> = ({
   );
 
   return isCasesContextValue(value) ? (
-    <QueryClientProvider client={casesQueryClient}>
```

Comment:
this had to be adjusted - otherwise using cases.ui ends up not finding theme in case where original plugin does not provide it - eg. Osquery plugin (previously had styled-components provider) using emotion provider, renders cases.ui addToCaseModal = it crashed because of missing theme.

Reply:
Thanks @tomsonpl! We need to fix it also in our side. Consumers of Cases, should not bother with this.